README.asciidoc

```diff
@@ -0,0 +1,401 @@
+# picr22-team-valdis
+
+Team members
+
+*Electronics:* Karl Sander Vinkel
+
+*Mechanics:* Karl Sander Vinkel
+
+*Programming:* Kevin Kirbits, Markus Männil, Arnold Leonhard Mertsin
+
+
+== Project Links
+
+https://a360.co/3goN0VZ[Valdis Fusion360]
+
+
+== Info
+
+Anydesk ip 198414610 - Valdis111
+
+SSH valdis@valdis password: valdis
+
+static ip: 172.17.152.9
+
+== Programming
+
+Libraries: 
+
+State Machine: 4 main states (FIND_BALL, MOVE_TO_BALL, ORBIT, THROW) FIND -> MOVE -> ORBIT -> THROW. Every tick robot decides in which state it's in from image processing
+
+opencv2: used for image processing
+
+pyrealsense2: for the camera implementation
+
+json: to work with the json data received from the referee
+
+websockets: To set up the client to receive the commands from the referee server
+
+math: for mathematical operations
+
+numpy: To work with the frames received from the image
+
+serial: to get access to the serial ports
+
+time: for time related tasks
+
+struct: Used to convert Python values to the motors. 
+
+Block Diagram: 
```

Comment:
Additionally to a block diagram add a basic description of your codes internal structure.